core/src/main/java/com/appintro/core/utils/TypefaceContainer.kt

```diff
@@ -1,4 +1,4 @@
-package com.github.appintro.internal
+package com.appintro.core.utils
 
 import android.graphics.Typeface
 import android.widget.TextView
```

Comment:
`TypefaceContainer` depends on `TextView`. Can you please move this as well in the view-specific module?